cmake/modules/AddSwift.cmake

```diff
@@ -721,6 +721,7 @@ function(_add_swift_library_single target name)
     swift_windows_generate_sdk_vfs_overlay(SWIFTLIB_SINGLE_VFS_OVERLAY_FLAGS)
     foreach(flag ${SWIFTLIB_SINGLE_VFS_OVERLAY_FLAGS})
       list(APPEND SWIFTLIB_SINGLE_SWIFT_COMPILE_FLAGS -Xcc;${flag})
+      list(APPEND SWIFTLIB_SINGLE_C_COMPILE_FLAGS ${flag})
```

Comment:
What is this for?

Reply:
Windows is case-insensitive, so when you're cross-compiling from a case-sensitive filesystem a number of header paths don't resolve (since the Windows headers are very loose about case and the spellings vary between files).
 
The VFS overlay is a means to map between the differently-cased forms of the headers (e.g. `windows.h` to `Windows.h`). Previously, it wasn't passing the argument to use the VFS overlay to the C/C++ runtime sources correctly (so it was working for Swift `import Msvcrt` only).